src/android/OAuthPlugin.java

```diff
@@ -129,6 +130,19 @@ public void onNewIntent(Intent intent) {
         }
     }
 
+  @Override
+  public void onResume(boolean multitasking) {
+    super.onResume(multitasking);
+    // Chrome Custom Tabs currently have no way to detect that they were closed.
+    // But if this happens, the app resumes and onResume is called which can be used as workaround.
+    // If we didn't handle a successful oAuth response, we can assume that the login was cancelled in onResume.
+    if(!loginSuccessful) {
+      this.webView.getEngine().evaluateJavascript("window.dispatchEvent(new MessageEvent('message', { data: 'oauth::{\"error\":\"cancelled\"}' }));", null);
+    } else {
+      // reset loginSuccessful to make sure it is correctly initialized on the next login attempt
+      loginSuccessful=false;
```

Comment:
Super minor nit, but formatting-wise, I prefer `if (statement)` with spaces, and spaces around the equals sign in assignments.

Reply:
Definitely on your side, fixed it to be consistent with the rest of the code :)